GEMB/managelayers.m

```diff
@@ -106,23 +110,10 @@
 EI(delete_cell)     = []; 
 EW(delete_cell)     = [];
 dzMin2(delete_cell) = []; 

```

Comment:
The `dzMin2` variable is no longer used after this, so no need delete cells from it.